stolon/README.md

```diff
@@ -0,0 +1,32 @@
+# stolon on Kontena
+
+[stolon](https://github.com/sorintlab/stolon/) is a cloud native PostgreSQL manager for PostgreSQL high availability. It's cloud native because it'll let you keep an high available PostgreSQL inside your containers.
+
+## Install
+> Prerequisites: You need to have working [Kontena](http://www.kontena.io) Container Platform installed. If you are new to Kontena, check [quick start guide](http://www.kontena.io/docs/getting-started/quick-start).
+
+```
+$ kontena volume create --driver local --scope instance stolon-keeper
+$ kontena stack install kontena/stolon
+```
+
+This will deploy stolon (PostgreSQL) cluster to your grid. Other services can connect to it with `stolon.${GRID}.kontena.local:5432` address. Stolon proxy at that address will make sure that client always connects to the database master instance.
+
+**Note:** Replace `${GRID}` with your grid name.
+
+## Administration
+
+Access `stolonctl` via interactive exec:
+
+```
+$ kontena service exec -it stolon/keeper bash
+root@keeper-1:/# stolonctl status --cluster-name $STKEEPER_CLUSTER_NAME --store-backend $STKEEPER_STORE_BACKEND --store-endpoints $STKEEPER_STORE_ENDPOINTS
```

Comment:
I thought that jumping into bash makes more sense because then user can execute also other  `stolonctl` commands.